test/dhtest.c

```diff
@@ -485,14 +491,19 @@ static const rfc5114_td rfctd[] = {
 static int run_rfc5114_tests(void)
 {
     int i;
-    for (i = 0; i < (int)OSSL_NELEM(rfctd); i++) {
-        DH *dhA, *dhB;
-        unsigned char *Z1 = NULL, *Z2 = NULL;
+    DH *dhA = NULL, *dhB = NULL;
+    unsigned char *Z1 = NULL, *Z2 = NULL;
+ 
+   for (i = 0; i < (int)OSSL_NELEM(rfctd); i++) {
```

Comment:
yeah, did't notice this at the previous commit, and now the indentation problem fixed:) thanks guys for your patient:)